### DISCUSSION TITLE
Instant filter based on selected value

### DISCUSSION THREAD

**Discussion by JrgMyr** [pawelsalawa/sqlitestudio#4413] [Ideas]
I would love to have a feature where I could select a field value and then be able to instantly filter the view/table by this value in this column.
This would mean a context menu entry should be created to put this value into "filter input" with exact matching (instead of partial matching).
I can do it now by double-clicking the value in question, then <Ctrl-C> to memorize it, then "show filter input", then <Ctrl-V> to input it, then adding '^' and '$', then switching to regular expressions, then activation. Just one context menu option do do all of that would be fantastic.

**Comment by pawelsalawa** [comment]
Created #4414 for it.
